### PR TITLE
preds.hard should be guaranteed to be an int

### PR DESCRIPTION
**Pull request by olliethomas**

### CODE REVIEW

File: ethicml/utility/data_structures.py [hunk 1]
```diff
@@ -240,7 +240,7 @@ class Prediction:
     def __init__(self, hard: pd.Series, info: Optional[Dict[str, float]] = None):
         """Make a prediction obj."""
         assert isinstance(hard, pd.Series), "please use pd.Series"
-        self._hard = hard
+        self._hard = hard.round().astype(int)
```
Comment (olliethomas) [inline ethicml/utility/data_structures.py:243]:
```suggestion
        self._hard = hard.apply(pd.to_numeric).round().astype(int)
```

**Comment by thomkeh** [comment]
Is the soft constructor already fine? https://github.com/predictive-analytics-lab/EthicML/blob/next/ethicml/utility/data_structures.py#L304

**Review by thomkeh — APPROVED** [review]
merge at will

**Comment by thomkeh** [comment]
Have you decided against this one? @olliethomas

**Comment by olliethomas** [comment]
> Have you decided against this one? @olliethomas

I think so... I think it's ugly

**Comment by thomkeh** [comment]
I see

**Pull request closed without merging**